examples/cifar10.json

```diff
@@ -1,7 +1,7 @@
 {
   "service": {
     "name": "cifar10",
-    "job_url": "https://s3-us-west-2.amazonaws.com/springle-tensorflow/slim.tar.gz",
+    "job_url": "https://downloads.mesosphere.io/tensorflow-dcos/examples/slim.tar.gz",
```

Comment:
For my own information, what is the difference between `mesosphere.io` and `mesosphere.com`?

Also, once we generate the universe package, the artifacts will be available at:
`https://downloads.mesosphere.com/tensorflow/assets/`, so I don't know if it would make sense to use `https://downloads.mesosphere.com/tensorflow/example/` as the examples root here?

We can also address this after we cut the universe package.

I am doing some final cleanup before I run those builds.